book/src/csi-driver-object.md

```diff
@@ -36,60 +36,63 @@ There are three important fields:
   * If a `CSIDriver` object does not exist for a given CSI Driver, the default is `true` -- meaning attach will be called.
   * If a `CSIDriver` object exists for a given CSI Driver, but this field is not specified, it also defaults to `true` -- meaning attach will be called.
   * For more information see [Skip Attach](skip-attach.md).
-* `podInfoOnMountVersion`
+* `podInfoOnMount`
   * Indicates this CSI volume driver requires additional pod information (like pod name, pod UID, etc.) during mount operations.
-  * If value is not specified, pod information will not be passed on mount.
-  * If value is set to a valid version, Kubelet will pass pod information as `volume_context` in CSI `NodePublishVolume` calls.
-  * Supported versions:
-    * Version "v1" will pass the following additional fields in `volume_context`:
-	  * `"csi.storage.k8s.io/pod.name": pod.Name`
-	  * `csi.storage.k8s.io/pod.namespace": pod.Namespace`
-	  * `csi.storage.k8s.io/pod.uid": string(pod.UID)`
+  * If value is not specified or `false`, pod information will not be passed on mount.
+  * If value is set to `true`, Kubelet will pass pod information as `volume_context` in CSI `NodePublishVolume` calls:
+    * `"csi.storage.k8s.io/pod.name": pod.Name`
+    * `csi.storage.k8s.io/pod.namespace": pod.Namespace`
```

Comment:
Looks like the opening `"` is missing for csi.storage.k8s.io/pod.namespace and csi.storage.k8s.io/pod.uid